Airdrop database migrations

rajeevbbqq opened the pull request.

raamb commented on alembic/versions/700e63da6309_airdroptables.py:
```diff
@@ -0,0 +1,119 @@
+    sa.Column('row_id', sa.INTEGER(), autoincrement=False, nullable=False),
+    sa.Column('address', sa.VARCHAR(length=50), nullable=False),
+    sa.Column('org_name', sa.VARCHAR(length=256), nullable=False),
+    sa.Column('token_name', sa.VARCHAR(length=50), nullable=False),
```
Lets increase token name to 128 characters

rajeevbbqq replied:
Updated in the latest commit @raamb

raamb commented on alembic/versions/700e63da6309_airdroptables.py:
```diff
@@ -0,0 +1,119 @@
+    sa.Column('row_id', sa.BIGINT(), autoincrement=True, nullable=False),
+    sa.Column('row_created', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('row_updated', mysql.TIMESTAMP(), server_default=sa.text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'), nullable=False),
+    sa.Column('airdrop_window_id', sa.INTEGER(), nullable=False),
```
Dont we need airdrop_id as well in this table?

rajeevbbqq replied:
I was thinking another way by leveraging with SQL queries we get the `airdrop _id` but that will be a heavy operation  @raamb  


Added the `airdrop_id` in the latest commit.

anandrgitnirman reviewed (commented):
General comments 
- Some of your tables are missing row_id , can we add this consistently across all tables 
- Can you also define the foreign key constraits
- define unique constraints on tables whereever appliacble

rajeevbbqq commented:
@anandrgitnirman 

- Every table has row_id, row_created, row_updated. Which is extending from AuditClass to each model for avoiding code duplications.
- Added foreign keys in the latest commit
- Added unique constraints in the latest commit

anandrgitnirman reviewed (commented):
lets also add created_date and updated_date on all the tables (where ever missing) , and make these as last columns ( to give a consistent feel)

rajeevbbqq commented:
Instead of `created_at, updated_at` we are using `row_created & row_updated` on every columns.
